Libraries/Experimental/SwipeableRow/SwipeableRow.js

```diff
@@ -200,61 +188,61 @@ const SwipeableRow = createReactClass({
         {swipeableView}
       </View>
     );
-  },
+  }
 
   close(): void {
-    this.props.onClose();
+    this.props.onClose && this.props.onClose();
```

Comment:
onClose is required based on the propTypes, you shouldn't need this check. If you do, then the flow types for onClose aren't properly showing that it is required.